lib/oauth.rb

```diff
@@ -0,0 +1,29 @@
+module Oauth
+	def self.get_oauth_token
+    response = HTTParty.post(ENV['ISSO_OAUTH_TOKEN_URL'],
+      body: {
+        grant_type: 'client_credentials',
+        client_id: ENV['ISSO_CLIENT_ID'],
+        client_secret: ENV['ISSO_CLIENT_SECRET']
+      })
+
+    case response.code
+    when 200
+      LogWrapper.log('INFO',
+        {
+        'message' => 'Token successfully received',
+        'statusCode'=> response.code,
```

Comment:
seems inconsistent -- statusCode here, but status below.  and I'm not actually sure of the syntax, but does the error level need to be passed in the json structure?